docs/changes/16903.other.rst

```diff
@@ -0,0 +1 @@
+Updated minimum supported Python version to 3.11. As a result, minimum required versions for ``ipython``, ``packaging``, ``pyarrow``, PyYAML, and ``scipy`` were updated as well to be compatible, while ``tomli`` and ``typing_extensions`` are no longer needed.
```

Comment:
As a user I would like optional and mandatory dependencies to be distinguished (we don't need to go into any details beyond that).

Reply:
`tomli` and `typing_extensions` were only needed for development, so they should not have been mentioned in the change log at all.

Reply:
I plan to follow up on this. Thanks !